tests/queries/0_stateless/03252_parse_datetime64_in_joda_syntax.reference

```diff
@@ -0,0 +1,14 @@
+2024-10-09 10:30:10.123
+2024-10-09 10:30:10.123456
+2024-10-10 02:30:10.123456
+2024-10-09 10:24:27.123456
```

Comment:
Have not read the code thoroughly, just a question:
Why `select parseDateTime64InJodaSyntax('2024-10-09 10:30:10.123456Asia/Shanghai', 'yyyy-MM-dd HH:mm:ss.SSSSSSz');` get result `2024-10-09 10:24:27.123456`

Reply:
The time offset computing between timezones is wrong, and now it is fixed.